pkg/oam/util/helper.go

```diff
@@ -197,3 +202,8 @@ func Object2Map(obj interface{}) (map[string]interface{}, error) {
 	err = json.Unmarshal(bts, &res)
 	return res, err
 }
+
+// GenTraitName generate trait name
+func GenTraitName(componentName string) string {
+	return fmt.Sprintf("%s-%s", TraitPrefixKey, componentName)
```

Comment:
I think @wonderflow has a good suggestion on the naming convention

Reply:
ok